src/test/java/guitests/guihandles/MainGuiHandle.java

```diff
@@ -49,4 +59,25 @@ public void use_APP_RESIZE_HOTKEY() {
         guiRobot.sleep(1000);
     }
 
+    public FxRobot sleepForGracePeriod() {
+        return guiRobot.sleep((ModelManager.GRACE_PERIOD_DURATION + 1), TimeUnit.SECONDS);
+    }
+
+    public FxRobot sleep(long duration, TimeUnit timeunit) {
+        return guiRobot.sleep(duration, timeunit);
+    }
```

Comment:
There is a TODO that states "//TODO: remove this from here, only *Handle objects should use the robot"
So there is no guiRobot in GuiTestBase now.